src/navigation/context.py

```diff
@@ -137,7 +137,7 @@ def other_gate_fid_pos(self) -> Optional[np.ndarray]:
         else:
             return None
 
-    def current_gate(self) -> Optional[Gate]:
+    def current_gate(self, odom_override: bool = True) -> Optional[Gate]:
```

Comment:
Can you just add a comment explaining that odom_override is true if we want to get it in odom frame in the doc string?